assignment/api/src/java/org/sakaiproject/assignment/api/model/Assignment.java

```diff
@@ -152,6 +153,9 @@
     @Column(name = "AUTHOR", length = 99)
     private String author;
 
+    @Transient
+    private String creatorOrModifier;
```

Comment:
This should not be added to a database POJO, this piece of data should be added as a var in the velocity context vs here.